devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -90,7 +91,7 @@ private getIkeaOnOffSwitchNames() {
 	]
 }
 
-private getIkeaOpenCloseRemoteNames() {
+private getIkeaOpenCloseOrSomfySituoRemoteNames() {
```

Comment:
Maybe just rename these to `getOpenCloseRemoteNames`. The function name is getting long.